cmd/bosun/conf/conf.go

```diff
@@ -283,10 +215,14 @@ func (ns *Notifications) Get(c *Conf, tags opentsdb.TagSet) map[string]*Notifica
 		if !ok {
 			continue
 		}
-		ns, err := c.parseNotifications(val)
-		if err != nil {
-			// Should already be checked by conf parser.
-			panic(err)
+		ns := make(map[string]*Notification)
+		for _, s := range strings.Split(val, ",") {
+			s = strings.TrimSpace(s)
+			n := c.GetNotification(s)
+			if n == nil {
+				continue // TODO error here?
```

Comment:
I think this is the correct behavior for now. But we will need to think about what happens if a config provider fails. All of those methods will likely need to return an error in the future.